.github/pull_request_template.md

```diff
@@ -0,0 +1,43 @@
+**Those changes are implementing:**
```

Comment:
this could be shortened to 
`**PR Type:** <sub>(fix/feature/rewrite/cleanup)</sub> `